### PR TITLE
#2694 Assessment should consider _isAvailable

### PR DESCRIPTION
**Pull request by jamesrea83**
https://github.com/adaptlearning/adapt_framework/issues/2694

### CODE REVIEW

File: js/assessment.js [hunk 1]
```diff
@@ -197,6 +197,7 @@ define([
       var states = {};
       for (var i = 0, l = this._assessments.length; i < l; i++) {
         var assessmentModel = this._assessments[i];
+        if (!assessmentModel.get('_isAvailable')) continue;
```
Comment (akshay-bhavsar) [inline js/assessment.js:200]: assessmentModel dosen't contains '_isAvailable'  property,
we need to first add it to assessmentModel

**Comment by akshay-bhavsar** [comment]
@jamesrea83 If you don't mind can i make changes??

File: js/assessment.js (hunk 1 above)
Comment (moloko) [reply js/assessment.js:200]: > assessmentModel dosen't contains '_isAvailable' property

@akshay-bhavsar it should do... certainly seems to when I check it.

image

Comment (akshay-bhavsar) [reply js/assessment.js:200]: @moloko  yes, i missed it.

my mistake!!

**Comment by akshay-bhavsar** [comment]
@moloko  is is possible???

https://github.com/adaptlearning/adapt-contrib-assessment/pull/124/files

File: js/assessment.js (hunk 1 above)
Comment (oliverfoster) [reply js/assessment.js:200]: That's the article model with the wrong variable name @akshay-bhavsar

**Pull request merged**